internal/characters/keqing/jump.go

```diff
@@ -0,0 +1,17 @@
+package keqing
+
+import (
+	"errors"
+
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/player"
+)
+
+func (c *char) Jump(p map[string]int) (action.Info, error) {
+	if c.Core.Status.Duration(player.XianyunAirborneBuff) > 0 {
```

Comment:
if c.StatusIsActive(player.XianyunAirborneBuff)

Reply:
not fixing for now since this is an example and should be deleted when xianyun gets implemented. chances are we should be checking individual char mods anyways